source/base/data_out_base.cc

```diff
@@ -6220,7 +6220,8 @@ write_pvd_record (std::ostream &out,
                   const std::vector<std::pair<double,std::string> >  &times_and_names) const
 {
   AssertThrow (out, ExcIO());
-
+   
+  out.precision(12);
```

Comment:
Yes, it certainly doesn't hurt to set it to 15 as well. Its just a number so, the amount of data is irrelevant. 

Yes, it makes more sense to set the precision to its original state.

Should I set it to 12 or 15 then ?